src/Confluent.Kafka/Admin/ElectLeadersOptions.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///     Options for ElectLeaders method.
```

Comment:
Rewrite as `Options for the "AdminClient.ElectLeaders" method.` to keep in sync with other classes